src/test/java/com/flightstats/hub/metrics/MetricsConfigTest.java

```diff
@@ -0,0 +1,54 @@
+package com.flightstats.hub.metrics;
```

Comment:
...so that was not my last comment, I guess... sorry!
This kind of looks like it's solely testing Lombok at this point. It might be worthwhile to borrow some code here for tests around the `InfluxdbReporterProvider`.

Reply:
No problem, you're right